docs/docsite/rst/dev_guide/testing/sanity/ignores.rst

```diff
@@ -86,6 +86,8 @@ Below are some example skip entries for an Ansible collection::
     plugins/module_utils/my_util.py validate-modules!skip # waiting for bug fix in module validator
     plugins/lookup/my_plugin.py compile-2.6!skip # Python 2.6 is not supported on the controller
 
+For a full list of sanity tests and test names, see :ref:`all_sanity_tests <https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/index.html#all-sanity-tests>`
```

Comment:
The ref should just refer to the page, rather than contain a full URL.

I'd probably go with the same text we have [elsewhere](https://github.com/ansible/ansible/blame/devel/docs/docsite/rst/dev_guide/testing_sanity.rst#L56) that references the list of sanity tests.

```suggestion
See the full list of :ref:`sanity tests <all_sanity_tests>`, which details the various tests and details how to fix identified issues.
```

Reply:
Noted, Thanks for your suggestion!